Changes

```diff
@@ -27,6 +27,10 @@ Working version
 
 ### Bug fixes:
 
+- #12207, #12222: Make closure computation linear in the number of recursive
+  functions instead of quadratic
+  (Vincent Laviron, report by François Pottier, review by ????)
```

Comment:
@lthls, can you add @Ekdohibs as reviewer here?

Reply:
Done